api/build.sh

```diff
@@ -1,20 +1,3 @@
-#
-# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
@moonming We may still need this header, right?

Reply:
Please add the ASF header.

Reply:
this is a mistake, should start with `#!/bin/sh` in this file, and the headers are following.
done